packages/ifpack2/src/Ifpack2_LinePartitioner_decl.hpp

```diff
@@ -80,8 +80,8 @@ class LinePartitioner : public OverlappingPartitioner<GraphType> {
   typedef typename GraphType::global_ordinal_type global_ordinal_type;
   typedef typename GraphType::node_type node_type;
   typedef Tpetra::RowGraph<local_ordinal_type, global_ordinal_type, node_type>  row_graph_type;
-  typedef Tpetra::MultiVector<Scalar,local_ordinal_type, global_ordinal_type, node_type>  multivector_type;
-  typedef typename Teuchos::ScalarTraits<Scalar>::magnitudeType MT;
+  typedef Tpetra::MultiVector<double,local_ordinal_type, global_ordinal_type, node_type>  multivector_type;
+  typedef typename Teuchos::ScalarTraits<double>::magnitudeType MT;
```

Comment:
The magnitudeType of double is always double.  If you meant to use MT only for coordinates, give it a more relevant name.  For example:
```
using coordinate_type = double;
```

Reply:
Yes, I decided to consistently switch everything to double.